Lib/fontTools/ttLib/ttFont.py

```diff
@@ -656,6 +656,17 @@ def getTableData(self, tag):
 		else:
 			raise KeyError(tag)
 
+	def getTableTags(self):
+		"""Returns a list of available table tags as strings.
+		"""
+		if self.reader:
+			return list(self.reader.keys())
```

Comment:
This misses tables that have been added to the font since it was read from disk.

Maybe just call `self.keys()` and delete `GlyphOrder` from the result?

Reply:
just do `return [tag for tag in self.keys() if tag != "GlyphOrder"]`

Reply:
Np!  Thanks for this input.  I'll change it.

Reply:
indeed, this getTableTabls looks to me just syntactic sugar given we already have `.keys()` and one can always filter "GlyphOrder" out with a one liner if one is not interested in it. But if you think it's still useful to have, I don't mind adding it

Reply:
> indeed, this getTableTabls looks to me just syntactic sugar given we already have `.keys()` and one can always filter "GlyphOrder" out with a one liner if one is not interested in it. But if you think it's still useful to have, I don't mind adding it

Sgtm.  Now I know the approach.  I'll use this in the future. Thanks all!